_posts/2021-04-19-Time Network Of Letters.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+author: Levi Blinder
+excerpt_separator: <!--more-->
+Title: Time Network Of Letters
+---
+
+[data set](https://docs.google.com/spreadsheets/d/1QALEFpTRY5cKbKZkRepm6U9iEKpJOvoPthxA8qA_uh8/edit?usp=sharing)
+(note that Sender column was used for source node, Recipient column for target node, and Date column for edge attribute, while the other two columns were not included)
+
+For each of three time periods (1500-1650, 1650-1800, and 1800-2000), I will show a view of the entire network as well as a closer view of a particularly interesting section of the network.
+
+All of these network pictures show a lot of data, so they are naturally a bit cluttered and it is hard to pick out individual connections. However, this does not mean they do not show qualitatively significant network features. Two such features that are worth looking at are:
+* The connections or lack thereof between components of the network, which can be observed from the broad view for each time period's network.
+* The connections or lack thereof between nodes (in particular those that are connected to a 'hub' node with a large number of adjacent edges) with few adjacent edges, which can be observed from the closer view for each time period's network.
+
+Broad view of letter network including letters from 1500-1650
```

Comment:
Both of these URLs link to the file held within your GitHub repository, not the image itself. For example, the URL for this image is https://github.com/leviBlinder/HUM331-Class-Repository/blob/master/images/CytoScape1500-1650Full.png?raw=true

- [ ] Before I can merge your post, you'll need to right click the images as they appear on your browser in GitHub and copy that address and replace the ones you've used here with the new ones.